Source/JavaScriptCore/dfg/DFGAbstractInterpreterInlines.h

```diff
@@ -1100,6 +1100,11 @@ bool AbstractInterpreter<AbstractStateType>::executeEffects(unsigned clobberLimi
             break;
         }
 
+        if (node->mustGenerate()) {
```

Comment:
Let's check `node->isBinaryUseKind(HeapBigIntUse) || node->isBinaryUseKind(AnyBigIntUse) || node->isBinaryUseKind(BigInt32Use)` here, since otherwise, we may have non bigint in right hand side.